README.md

```diff
@@ -8,41 +9,48 @@ In Custom Software we strive for software excellence.
 Therefore, we value what clever people have done for research into creating more robust and maintainable software. 
 This results in the following principles for how we work.
 
-* We do extensive review of everything
+* We do extensive reviews of all things
 * We test everything, preferably through automation
 * We code defensively
 * We take ownership of the full build and release pipeline for both code and platform
+* We strive for communication to be thorough and constructive, rather than quick and dirty
```

Comment:
If we're adding a point here, should we also add a clarification section below?

(fwiw we should probably also make them links, but that can be another PR)